firmware/ec/test/include/test_led.h

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+#ifndef _TEST_LED_H
+#define _TEST_LED_H
+
+#include "fake/fake_GPIO.h"
+#include "fake/fake_I2C.h"
+#include "inc/devices/led.h"
+#include "inc/devices/sx1509.h"
+#include "include/test_sx1509.h"
+#include <string.h>
+#include <ti/sysbios/knl/Task.h>
+#include "unity.h"
+
```

Comment:
Can we order the macros based on values

Reply:
No, We are arranging macros in alphabetical order to increase the readability.